book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
```

Comment:
- [ ] Great job so far. please note that you are to create the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`.  please ensure Getters and Setters for @rentals should be present 
`attr_accessor :title, :author, :rentals`